### PR TITLE
Persist converted pdf and publish it as pdf/dump.pdf and @@pdf

### PR DESCRIPTION
**Pull request by jaroel**
This allows uploading a docx and displaying a pdf version of it using pdf.js

### CODE REVIEW

File: collective/documentviewer/browser/configure.zcml [hunk 1]
```diff
@@ -86,4 +86,11 @@
     attribute="getTranslatedJSLabels"
     permission="zope.Public" />
 
+  <browser:page
+    for="*"
+    name="dv_pdf_url"
```
Comment (tisto) [inline collective/documentviewer/browser/configure.zcml]: @jaroel I would like to use a less obscure URL here. Can't we just use pdf?

**Comment by tisto** [comment]
@tkimnguyen @vangheem who can we ask for a pull request review here? Anybody who could merge and do a release? If you don't have time, you could also consider granting us permissions to do a release on pypi (my username is timo).

**Comment by tisto** [comment]
FYI: We need this for an Angular 2 client that just uses c.documentviewer internally to convert other formats to PDF. Our font-end document viewer can just render PDF.

**Comment by vangheem** [comment]
@tisto I've added you as a maintainer on pypi.

Feel free to merge when you feel good about the current question of browser view name.

File: collective/documentviewer/browser/configure.zcml (hunk 1 above)
Comment (jaroel) [inline collective/documentviewer/browser/configure.zcml]: @tisto sure, I've changed it!

**Comment by tisto** [comment]
@vangheem thank you!

**Comment by tisto** [comment]
@jaroel we might want to merge this feature into the Plone 5 version of c.documentviewer. Though, we don't need that for our current project...

**Comment by jaroel** [comment]
@tisto I've added an issue for that: #65

**Pull request merged**